### PR TITLE
[Aruba] Add DHCPv6 Relay and Snooping events

### PR DESCRIPTION
**Pull request by dwhyrock**
Issue: https://github.com/elastic/integrations/issues/11542

This PR adds in logs for HPE Aruba DHCPv6 Relay and Snooping Events.  The Relay events needed no extra processing.

The logs were hand-crafted based on HPE's page: 
 * https://www.arubanetworks.com/techdocs/AOS-CX/10.07/HTML/5200-8214/Content/events/DHCPv6-RELAY.htm
 * https://www.arubanetworks.com/techdocs/AOS-CX/10.07/HTML/5200-8214/Content/events/DHCPv6-SNOOPING.htm

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [ ] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## Author's Checklist

- [ ] 

## Related issues

-

### CODE REVIEW

File: packages/hpe_aruba_cx/_dev/build/docs/README.md [hunk 1]
```diff
@@ -224,16 +224,23 @@ Note: Descriptions have not been filled out
...
-| aruba.dhcp.ipv6_address  |             |      | client.ip          |
-| aruba.dhcp.mac           |             |      | client.mac         |
-| aruba.dhcp.new_port      |             |      |                    |
-| aruba.dhcp.port          |             |      | server.port        |
-| aruba.dhcp.vid           |             |      | network.vlan.id    |
-| aruba.dhcp.volume_name   |             |      |                    |
+
+| Doc Fields                     | Schema Mapping       |
+|--------------------------------|----------------------|
+| <event>                        | event.code           |
```
Comment (qcorporation) [inline packages/hpe_aruba_cx/_dev/build/docs/README.md:230]: I think we use the `event.code` for the header, it doesn't look like there's an <event> in the docs

File: packages/hpe_aruba_cx/_dev/build/docs/README.md [hunk 2]
```diff
@@ -224,16 +224,23 @@ Note: Descriptions have not been filled out
...
+| <severity>                     | log.level            |
+| <message>                      | message              |
+| <category>                     | event.category       |
+| <description>                  | event.reason         |
+| <ipv6_address>                 | client.ip            |
```
Comment (qcorporation) [inline packages/hpe_aruba_cx/_dev/build/docs/README.md:235]: 8301 is Server <ipv6_address> packet received on untrusted port <port> dropped.
do we want to use `client.ip` or should we use `server.ip`

File: packages/hpe_aruba_cx/_dev/build/docs/README.md [hunk 3]
```diff
@@ -224,16 +224,23 @@ Note: Descriptions have not been filled out
...
+| <port>                         | server.port          |
+| <mac>                          | client.mac           |
+| <existing_port>                | server.port          |
```
Comment (qcorporation) [inline packages/hpe_aruba_cx/_dev/build/docs/README.md:238]: I think you are using `aruba.port`

File: packages/hpe_aruba_cx/_dev/build/docs/README.md [hunk 4]
```diff
@@ -224,16 +224,23 @@ Note: Descriptions have not been filled out
...
+| <new_port>                     | server.port          |
+| <ip>                           | client.ip            |
+| <vid>                          | network.vlan.id      |
+| <volume_name>                  | file.name            |
```
Comment (qcorporation) [inline packages/hpe_aruba_cx/_dev/build/docs/README.md:242]: looks like you are using `aruba.dhcp.volume_name`

File: packages/hpe_aruba_cx/_dev/build/docs/README.md [hunk 5]
```diff
@@ -224,16 +224,23 @@ Note: Descriptions have not been filled out
...
+| <event>                        | event.code           |
+| <severity>                     | log.level            |
+| <message>                      | message              |
+| <category>                     | event.category       |
+| <description>                  | event.reason         |
+| <ipv6_address>                 | client.ip            |
+| <port>                         | server.port          |
+| <mac>                          | client.mac           |
+| <existing_port>                | server.port          |
+| <new_port>                     | server.port          |
```
Comment (qcorporation) [inline packages/hpe_aruba_cx/_dev/build/docs/README.md:239]: `new_port` seems to be mapped to `aruba.dhcp.new_port`

**Review by qcorporation — COMMENTED** [review]
everything looks OK, docs just need to be updated with the proper fields.

One question I have is do we want to rely on the period at the end of the log message sentence. I think I've seen other instances where we are more liberal with the parsing and some instances like this one where we are more rigid

File: packages/hpe_aruba_cx/_dev/build/docs/README.md (hunk 1 above)
Comment (dwhyrock) [reply packages/hpe_aruba_cx/_dev/build/docs/README.md:230]: i'll remove it from here

File: packages/hpe_aruba_cx/_dev/build/docs/README.md (hunk 2 above)
Comment (dwhyrock) [reply packages/hpe_aruba_cx/_dev/build/docs/README.md:235]: IMO, `<ipv6_address>` could be either `client.ip` or `server.ip`, depending on which message it is.  For `8301`, it should be `server.ip`, but for others when talking about a node and its IP address that it was assigned, it should be `client.ip`.  I'll put both in the docs here.

**Comment by dwhyrock** [comment]
> One question I have is do we want to rely on the period at the end of the log message sentence. I think I've seen other instances where we are more liberal with the parsing and some instances like this one where we are more rigid

We haven't been very consistent.  I'm happy either way.  If we want to remove the periods at the end of the grok statements, I say that should be in a PR of its own.

We also have a bunch of these in the expected json documents that shouldn't actually be set:

```
"aruba": {
    "sequence": "-"
}
```

If the `sequence` field is just a `-`, it shouldn't be there, IMO.  That should also be its own PR (maybe combined with the no-period one).

**Review by qcorporation — APPROVED** [review]
thanks for the changes - approved

**Review by gogochan — APPROVED** [review]

**Pull request merged**